qiskit/circuit/quantumcircuit.py

```diff
@@ -960,22 +960,36 @@ def width(self):
         return sum(reg.size for reg in self.qregs + self.cregs)
 
     @property
-    def n_qubits(self):
-        """
-        Return number of qubits.
-        """
+    def num_qubits(self):
+        """Return number of qubits."""
         qubits = 0
         for reg in self.qregs:
             qubits += reg.size
         return qubits
 
     @property
-    def n_clbits(self):
-        """
-        Return number of classical bits.
-        """
+    def n_qubits(self):
+        """Deprecated, use ``num_qubits`` instead. Return number of qubits."""
+        warnings.warn('The QuantumCircuit.n_qubits method is deprecated as of 0.14.0, and '
```

Comment:
The next release is currently 0.13.0? Are we going to defer this until after 0.13.0, if so then we should label this on hold. Otherwise lets change this to 0.13.0

Reply:
Yes, you're right